packages/ui-toggle-details/src/ToggleGroup/index.tsx

```diff
@@ -162,6 +174,7 @@ class ToggleGroup extends Component<ToggleGroupProps> {
               display="block"
               borderRadius="medium"
               background="primary"
+              themeOverride={{ borderColorPrimary: styles?.borderColor }}
```

Comment:
we shouldn't use `themeOverride` for our own components

Reply:
removed